src/index/hgraph_index.cpp

```diff
@@ -78,8 +78,8 @@ HGraphIndex::Init() {
 
     mult_ = 1 / log(1.0 * static_cast<double>(this->bottom_graph_->MaximumDegree()));
 
-    this->pool_ = std::make_shared<hnswlib::VisitedListPool>(
-        1, this->bottom_graph_->MaxCapacity(), allocator_);
+    this->pool_ =
```

Comment:
no need using Allocator ?

Reply:
I will change the memory allocated by make_shared to be allocated using an allocator in the subsequent PR